### PR TITLE
Updating deps for ex_doc, inch_ex

### PR DESCRIPTION
**Pull request by sger**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -17,6 +17,9 @@ defmodule Kitto.Mixfile do
        "coveralls.post": :test,
        "coveralls.travis": :test,
        "coveralls.html": :test],
+     name: "kitto",
```
Comment (zorbash) [inline mix.exs]: Let's Capitalize this.

**Comment by Zorbash** [comment]
Hello, @sger! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/kittoframework/kitto/pulls/44). Please check our [documentation](https://ebertapp.io/docs) for more information.

File: mix.exs (hunk 1 above)
Comment (sger) [reply mix.exs]: :+1:

File: mix.exs [hunk 2]
```diff
@@ -30,12 +33,11 @@ defmodule Kitto.Mixfile do
      {:plug, "~> 1.2"},
      {:poison, "~> 3.0"},
      {:fs, github: "kittoframework/fs", tag: "v1.9-kitto-1"},
-     {:ex_doc, ">= 0.0.0", only: :dev},
+     {:ex_doc, "~> 0.14", only: :dev},
```
Comment (zorbash) [inline mix.exs:36]: You should also commit changes introduced in `mix.lock`.

Comment (sger) [reply mix.exs:36]: @Zorbash oops forgot it 
`mix deps.unlock --all`
`mix deps.get`

maybe adding in a new PR typespecs and behaviours http://elixir-lang.org/getting-started/typespecs-and-behaviours.html

Comment (zorbash) [reply mix.exs:36]: No need to unlock them all at the time. Not enough time to go through the changelog of each of the updated deps.

Run `mix deps.get` and commit the changes inflicted by your updates in `mix.exs`.

**Comment by zorbash** [comment]
Closed via: https://github.com/kittoframework/kitto/commit/f5bf1f19f5fbea32bbf28eadaa7af51d2593aa2c

Thanks @sger!

**Pull request closed without merging**